packages/builder/src/stores/backend/tables.js

```diff
@@ -63,6 +80,19 @@ export function createTablesStore() {
     const savedTable = await API.saveTable(updatedTable)
     replaceTable(savedTable._id, savedTable)
     select(savedTable._id)
+    // make sure tables up to date (related)
+    const tableIdsToFetch = []
+    for (let column of Object.values(updatedTable?.schema || {})) {
+      if (column.type === FIELDS.LINK.type) {
+        tableIdsToFetch.push(column.tableId)
+      }
+    }
```

Comment:
Might be worth throwing in a wee:
```js
tableIdsToFetch = [...new Set(tableIdsToFetch)]
```
Just to ensure we don't have any duplicates - since it's possible that you have 2 relationships to the same table I guess.